integration/cluster_api_endpoints_test.go

```diff
@@ -0,0 +1,220 @@
+// +build integration
+
+package integration_test
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	awseks "github.com/aws/aws-sdk-go/service/eks"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	. "github.com/weaveworks/eksctl/integration/matchers"
+	. "github.com/weaveworks/eksctl/integration/runner"
+
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+)
+
+const (
+	createCluster    = `CreateCluster`
+	updateCluster    = `UpdateCluster`
+	endpointPubTmpl  = `EndpointPublicAccess: %v`
+	endpointPrivTmpl = `EndpointPrivateAccess: %v`
+)
+
+var False = false
```

Comment:
usages of this should be replaced by `Disabled()`

Reply:
Done.